config.schema.json

```diff
@@ -126,123 +126,663 @@
               }
             },
             "command1": {
-              "title": "Custom Area 1",
+              "title": "Spot Area 1",
               "type": "string",
-              "description": "Enter a 'Spot Area' number or multiple numbers separated by a comma.",
+              "description": "Enter a spot area ID or multiple IDs separated by a comma.",
               "condition": {
-                "functionBody": "return (model?.devices?.[arrayIndices]?.deviceId?.length > 0 && !model?.devices?.[arrayIndices]?.ignoreDevice);"
+                "functionBody": "return (model?.devices?.[arrayIndices]?.deviceId?.length > 0 && !model?.devices?.[arrayIndices]?.ignoreDevice  && model?.devices?.[arrayIndices]?.command1);"
               }
             },
             "command2": {
-              "title": "Custom Area 2",
+              "title": "Spot Area 2",
               "type": "string",
-              "description": "Enter a 'Spot Area' number or multiple numbers separated by a comma.",
+              "description": "Enter a spot area ID or multiple IDs separated by a comma.",
               "condition": {
-                "functionBody": "return (model?.devices?.[arrayIndices]?.deviceId?.length > 0 && !model?.devices?.[arrayIndices]?.ignoreDevice && model?.devices?.[arrayIndices]?.command1?.length > 0);"
+                "functionBody": "return (model?.devices?.[arrayIndices]?.deviceId?.length > 0 && !model?.devices?.[arrayIndices]?.ignoreDevice && model?.devices?.[arrayIndices]?.command2);"
```

Comment:
again the check above and for the rest of the commands below

Reply:
See above.